src/diffusers/loaders.py

```diff
@@ -225,6 +230,19 @@ def load_attn_procs(self, pretrained_model_name_or_path_or_dict: Union[str, Dict
         is_custom_diffusion = any("custom_diffusion" in k for k in state_dict.keys())
 
         if is_lora:
+            is_new_lora_format = all(
+                key.startswith(self.unet_name) or key.startswith(self.text_encoder_name) for key in state_dict.keys()
+            )
+            if is_new_lora_format:
+                raise ValueError(
+                    "You are using the new LoRA serialization format introduced in https://github.com/huggingface/diffusers/pull/2918. Please use `pipe.load_lora_weights(...)`"
+                )
```

Comment:
Arises when someone serializes with the new LoRA format and then tries to load with `pipe.unet.load_attn_procs()`.